### ISSUE TITLE
Crash in development mode. Leanplum.start() can't handle JS refreshes.

### ISSUE THREAD

**Issue by defualt** [Leanplum/Leanplum-ReactNative-SDK#5]
In my React Native application, `Leanplum.start()` is called whenever the app's javascript refreshes without reloading the entire app in the course of routine development.

Upon refresh after I make a js code change, the entire app crashes, and the Xcode log says: 
`Error', reason: 'Leanplum: Already called start. This error is only thrown in development mode.'`

There is no way to avoid calling `Leanplum.start()`.  I can't detect if it's already running.

There is no `stop` method in the `Leanplum-ReactNative-SDK` so I can't stop it before starting it again to possible avoid the error.

I'm afraid this same crash would occur production in combination with CodePush which also entails a JS refresh.

**Comment by barsumek** [comment]
+1, this issue blocks using the SDK on Dev environment and is error-prone on Prod.

Android version of the SDK also logs this error, but it does not crash the app. I'm not sure if it causes any side-effects or not, though.

Both SDK versions should be resilient to JS reloads as it's the core of React Native development.

**Comment by defualt** [comment]
To work in dev, add this ugliness to your pod file until the library is fixed.
```  
  post_install do |installer|
    system("sed -i '' -e \"s/if (state.calledStart) {/if (state.calledStart\\/* postinstall *\\/) {[Leanplum resume];return;/g\" ./Pods/Leanplum-iOS-SDK/Leanplum-SDK/Classes/Internal/Leanplum.m")
  end
```

**Comment by e7mac** [comment]
Fixed in [v1.0.11](https://github.com/Leanplum/Leanplum-ReactNative-SDK/releases/tag/1.0.11)
